Winapp3/Winapp3.ini

```diff
@@ -235,50 +180,56 @@ ExcludeKey31=PATH|%CommonAppData%\Microsoft\Search\Data\Applications\Windows\|*.
+ExcludeKey46=PATH|%SystemDrive%\Users\*\AppData\Local\Packages\Microsoft.*Cortana_*\AppData\Indexed DB\|*.chk;*.log
+ExcludeKey47=PATH|%SystemDrive%\Users\*\AppData\Local\Packages\Microsoft.*Cortana_*\LocalState\|speech_onecorereg.bin.LOG*
+ExcludeKey48=PATH|%SystemDrive%\Users\*\AppData\Local\Packages\Microsoft.*Cortana_*\LocalState\DeviceSearchCache\|*.txt
+ExcludeKey49=PATH|%SystemDrive%\Users\*\AppData\Local\Packages\Microsoft.MicrosoftEdge_*\AC\MicrosoftEdge\User\Default\DataStore\Data\*\*\DBStore\|*.chk
+ExcludeKey50=PATH|%SystemDrive%\Users\*\AppData\Local\TileDataLayer\Database\|EDB*.*
+ExcludeKey51=PATH|%SystemDrive%\Users\*\AppData\Roaming\steelseries-engine-*-client\Local Storage\leveldb\|*.log
+ExcludeKey52=PATH|%SystemDrive%\Users\All Users\Microsoft\Search\Data\Applications\Windows\|*.chk;*.log
+ExcludeKey53=PATH|%WinDir%\appcompat\Programs\|*.LOG*
+ExcludeKey54=PATH|%WinDir%\ServiceProfiles\LocalService\|NTUSER.*LOG*
+ExcludeKey55=PATH|%WinDir%\ServiceProfiles\NetworkService\|NTUSER.*LOG*
+ExcludeKey56=PATH|%WinDir%\ServiceProfiles\NetworkService\AppData\Local\Microsoft\Windows\DeliveryOptimization\State\|*.LOG*
+ExcludeKey57=PATH|%WinDir%\SoftwareDistribution\DataStore\Logs\|edb*.*
+ExcludeKey58=PATH|%WinDir%\System32\config\|BBI.LOG*
+ExcludeKey59=PATH|%WinDir%\System32\config\|BCD-Template.LOG*
+ExcludeKey60=PATH|%WinDir%\System32\config\|COMPONENTS.LOG*
+ExcludeKey61=PATH|%WinDir%\System32\config\|DEFAULT.LOG*
+ExcludeKey62=PATH|%WinDir%\System32\config\|DRIVERS.LOG*
+ExcludeKey63=PATH|%WinDir%\System32\config\|ELAM.LOG*
+ExcludeKey64=PATH|%WinDir%\System32\config\|SAM.LOG*
+ExcludeKey65=PATH|%WinDir%\System32\config\|SECURITY.LOG*
+ExcludeKey66=PATH|%WinDir%\System32\config\|SOFTWARE.LOG*
+ExcludeKey67=PATH|%WinDir%\System32\config\|SYSTEM.LOG*
+ExcludeKey68=PATH|%WinDir%\System32\config\|userdiff.LOG*
+ExcludeKey69=PATH|%WinDir%\System32\config\bbimigrate\|*.LOG*
+ExcludeKey70=PATH|%WinDir%\System32\config\RegBack\|*.LOG*
+ExcludeKey71=PATH|%WinDir%\System32\config\systemprofile\|NTUSER.*LOG*
+ExcludeKey72=PATH|%WinDir%\System32\SMI\Store\Machine\|SCHEMA.DAT.LOG*
+ExcludeKey73=PATH|%WinDir%\SysWOW64\config\|DEFAULT.*LOG*
+ExcludeKey74=PATH|%WinDir%\SysWOW64\config\|SAM.*LOG*
+ExcludeKey75=PATH|%WinDir%\SysWOW64\config\|SECURITY.*LOG*
+ExcludeKey76=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.*LOG*
+ExcludeKey77=PATH|%WinDir%\SysWOW64\config\systemprofile\|NTUSER.*LOG*
+ExcludeKey78=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.chk;*.log
+ExcludeKey79=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\Microsoft\Windows\WebCache\|*.chk
+
+[Registry Hive Log Files *]
 Section=Dangerous Long
 Detect=HKLM\Software\Microsoft
-Default=False
-Warning=You should not delete these files if they are in a Chrome/Chromium profile.
-FileKey1=%SystemDrive%|*.LOG|RECURSE
+FileKey1=%SystemDrive%|*.blg;*.log1;*.log2;.regtrans-ms|RECURSE

```

Comment:
Minor correction required: It should be "\*.blf" (typo) and "\*.regtrans-ms" (missing asterisk).